docs/src/create-apps/runtime-operations.md

```diff
@@ -0,0 +1,333 @@
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 ping $APP
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-ping", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Reload your app
++++
+
+To reload your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-reload:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 reload $APP
+``` 
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-reload", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+<--->
++++
+title=Restart your app
++++
+
+To restart your Node.js app, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+    pm2-restart:
+        role: admin 
+        commands: 
+            start: |
+                # Assuming pm2 start npm --no-daemon --watch --name $APP -- start -- -p $PORT
+                APP=$(cat package.json | jq -r '.name')
+                pm2 restart $APP 
+``` 
+
+To trigger your runtime operation, run a command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "pm2-restart", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+{{< /codetabs >}}
+
+### Define management commands on your Django project
+
+On a Django project, you can [define custom `django-admin` commands](https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/), for example to run a migration outside of the Django ORM.
+To do so, define a runtime operation similar to the following:
+
+```yaml {location=".platform.app.yaml"}
+operations:
+  manual-migration:
+    role: admin
+    commands:
+      start: python manage.py manual_migration
+```
+
+To trigger your runtime operation, run a cURL command similar to the following:
+
+```bash
+$ platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/deployments/current/operations -X POST -d '{"operation": "manual-migration", "service": "{{< variable "CONTAINER_NAME" >}}"}' -p {{< variable "PROJECT_ID" >}}
+```
+
+To show you how much effort using such a runtime operation saves you,
```

Comment:
I don't get this point and the following python code. It's not the code of the command `manual_migration`. We can probably remove it, and Django user will automatically get what does a `manage.py`.